lua/nvim-tree.lua

```diff
@@ -88,6 +88,7 @@ end
 
 function M.open(cwd)
   cwd = cwd ~= "" and cwd or nil
+  vim.g.nvim_tree_tab_open = 1
```

Comment:
We do not use global variables to track nvim-tree state.

Please test via something like:
* iterate windows on tab
* test `utils.is_nvim_tree_buf`